frontend/vite.config.js

```diff
@@ -48,6 +50,46 @@ function gitVersionPlugin() {
   };
 }
 
+function faviconPlugin(originalPath) {
+  const files = {};
+  function addFiles(ctx, fileName, source) {
+    ctx.emitFile({ type: 'asset', fileName, source });
+    files[`/${fileName}`] = source;
+  }
+
+  return {
+    name: 'favicon-plugin', // required, will show up in warnings and errors
+    async buildStart() {
+      const response = await favicons(originalPath, {
+        icons: {
+          android: false,
+          appleIcon: false,
+          appleStartup: false,
+          windows: false,
+          yandex: false,
+
+          favicons: true,
+        },
+      });
+
+      addFiles(this, "favicon.svg", fs.readFileSync(originalPath))
+      response.images.forEach(img => {
+        addFiles(this, img.fileName, img.contents)
```

Comment:
```suggestion
        addFiles(this, img.file, img.contents)
```

`FaviconFile` has no attribute `fileName`